style.css

```diff
@@ -43,21 +53,228 @@ body {
   flex-direction: column;
 }
 
-ul {
+.socials-mainlist {
   display: flex;
   padding: 0;
   margin-top: 10px;
 }
 
-ol {
+.socials-list {
   padding: 0;
   padding-right: 20px;
 }
 
-h4 {
+.letsconnect {
+  width: 160px;
+  height: 24px;
   color: #7f8cff;
+  font-size: 20px;
 }
 
 .logo {
   color: #6070ff;
 }
+
+.aboutme-intro {
+  color: #344563;
+}
+
+.template-aboutme {
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-color: #fff;
+  padding: 100px 5%;
+  border-radius: 0 0 0 100px;
+  position: relative;
+  left: 0;
+  top: 270px;
+}
+
+.resume {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background-color: #fff;
+  border-radius: 15px 15px 15px;
+  padding: 0.75em;
+  width: 170px;
+  height: 50px;
+  margin-top: 5px;
+  margin-bottom: 30px;
+}
```

Comment:
The same thing I mentioned for the other button, this could look/behave even better and offer a better UX/UI. 

## Figma 
![Screenshot 2023-02-17 at 21 00 33](https://user-images.githubusercontent.com/45739963/219825809-881ae858-ffd6-456f-b3aa-f1bbeb9d3b2e.png)

## Your design

![Screenshot 2023-02-17 at 21 01 08](https://user-images.githubusercontent.com/45739963/219825842-cfaecd94-f6e6-4b2f-8238-4e212331e843.png)

Also, make sure the button follows the button styling: Active, Hover, and Pressed (see below 👇🏽 )

![Screenshot 2023-02-17 at 20 52 56](https://user-images.githubusercontent.com/45739963/219825480-90aea75f-5322-4e74-b6c2-5a8ebc18523b.png)